src/hooks/data/useVaultApy.ts

```diff
@@ -1,5 +1,23 @@
+import { useQueries } from "@tanstack/react-query"
+import { useContractRead, useQuery } from "wagmi"
```

Comment:
Does it work to import `useQuery` from wagmi?

Reply:
yes